junebug/config.py

```diff
@@ -34,3 +34,7 @@ class JunebugConfig(Config):
             'username': 'guest',
             'password': 'guest'
         })
+
+    ttl = ConfigInt(
+        "Maximum time allowed to reply to messages",
+        default=60)
```

Comment:
Since we are going to have a ttl for storing outbound messages, maybe we should name this to `inbound_ttl`?